libsolidity/ast/Types.cpp

```diff
@@ -110,6 +110,14 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 	return transformed;
 }
 
+string signatureParameterList(TypePointers const& _types, bool _short)
```

Comment:
If you look at the implementation of this function, there is no "parameter" to be found, yet it is in the name of the function.
Maybe `toStringInParentheses` would be a better name?

Reply:
Done.

Reply:
Agree, done.

Reply:
> there is no "parameter" to be found, yet it is in the name of the function.

I suggested that name because that's the intended usage and the function makes some small assumptions related to that (like the lack of space after the comma; I originally wanted to skip `_short` too but we need it after all). It's not really a generic `toString()` so I think it's misleading to call it that.

But ok, let's leave it at `toStringInParentheses()` for now to avoid going back and forth on this.